README.md

```diff
@@ -22,7 +22,8 @@ release.
 
 | version                                                                                                 | planned end of life date |
 | ------------------------------------------------------------------------------------------------------- | ------------------------ |
-| [v3.16](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v3.16/docs/README.md) | TBA                      |
+| [v4.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v4.0/docs/README.md)   | TBA                      |
+| [v3.16](https://github.com/SumoLogic/sumologic-kubernetes-collection/tree/release-v3.16/docs/README.md) | 2024-10-20               |
```

Comment:
are we going to support v3 for a year since today?

Reply:
Good catch!